starter-code/scripts/articleView.js

```diff
@@ -60,6 +75,16 @@ articleView.setTeasers = function() {
 
     // STRETCH GOAl!: change the 'Read On' link to 'Show Less'
   */
+  $('.read-on').on('click', function(event) {
+    event.preventDefault();
+    $(this).prev().find('*').show();
```

Comment:
Good work here! I know this was wonky in lab and I'm glad that we got it working. Dynamic DOM manipulation can be weird sometimes :)